### ISSUE TITLE
Request: Allow custom implementation without attribute

### ISSUE THREAD

**Issue by sellig** [mikependon/RepoDB#399]
so it is what allows us to keep the POCOs attribute free.
as in As FluentMap à la https://dapper-tutorial.net/dapper-fluentmap
or  https://github.com/tmsmith/Dapper-Extensions/wiki/AutoClassMapper

Brainstorm implementation
```csharp
TypeMapper.RegisterMap(new CustomerMap());
 
public class CustomerMap :  EntityMap<Customer>
{
    public CustomerMap()
    {
        Map("[dbo].[Customer]");
        
	Map(p => p.Id).IsKey();

	Map(p => p.Id).IsIdentity();

        Map(p => p.Id)
            .ToColumn("CustomerId");

        Map(p => p.ExtraInfo)
            .UsePropertyHandler<CustomerExtraInfoHandler>();

       Map(p => p.Image)
            .UseMicrosoftSqlServerDbType(SqlDbType.Binary);
    }
}
```

**Comment by mikependon** [comment]
Thank you for this report. This is very good feature. We will assess and will update this ticket soon.

**Comment by mikependon** [comment]
@sellig - this is now deployed at [RepoDb (v1.11.0-beta1)](https://www.nuget.org/packages/RepoDb/1.11.0-alpha1). Can you test it for me?

Please refer the comment to the other thread we had #415.

**Comment by mikependon** [comment]
This request of yours can be addressed by `FluentMapper` class as part of the latest release.

```csharp
FluentMapper
	.Entity<Customer>() // Which Entity Model?
	.Table("[sales].[Customer]") // Table/View Mapping
	.Primary(e => e.Id) // Primary Mapping
	.Identity(e => e.Id) // Identity Mapping
	.Column(e => e.Id, "CustomerId") // Column Mapping
	.Column(e => e.Name, "CustomerName") // Column Mapping
	.DbType(e => e.DateOfBirth, DbType.DateTime2) // DbType Mapping
	.PropertyHandler<CustomerAddressPropertyHandler>(e => e.Address); // PropertyHandler Mapping
```

The feature is available at [RepoDb (v1.11.0)](https://www.nuget.org/packages/RepoDb/1.11.0).

```csharp
> Install-Package RepoDb -version 1.11.0
```

Please visit the [Implicit Mapping](http://repodb.net/feature/implicitmapping) to learn more.
